### ISSUE TITLE
Ability to navigate to previous or next record when in expand indicator record

### ISSUE THREAD

**Issue by andrewtpham** [hikaya-io/hakawati#411]

**Comment by michaelbukachi** [comment]
@andrewtpham  How does one get this dialog? I tried to use expand record when right-clicking but it didn't work.

**Comment by andrewtpham** [comment]
@michaelbukachi you're right it seems broken. I've asked @Kimaiyo077 if he knows about the issue. I dont see any errors in the console or network tabs.
